SemanticFormsSelect.php

```diff
@@ -6,18 +6,27 @@
  *
  * @defgroup SFS SemanticFormsSelect
  */
-if ( !defined( 'MEDIAWIKI' ) ) {
-	die( 'This file is part of the SemanticFormsSelect extension, it is not a valid entry point.' );
-}
 
-if ( version_compare( $GLOBALS[ 'wgVersion' ], '1.23', 'lt' ) ) {
+if ( version_compare( $GLOBALS[ 'wgVersion' ], '1.23c', 'lt' ) ) {
 	die( '<b>Error:</b> This version of <a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">SemanticFormsSelect</a> is only compatible with MediaWiki 1.23 or above. You need to upgrade MediaWiki first.' );
 }
 
-if ( !defined( 'SF_VERSION' ) || version_compare( SF_VERSION, '2.8', 'lt' ) ) {
-   die( '<b>Error:</b> This version of <a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">SemanticFormsSelect</a> is only compatible with Semantic Forms 2.8 or above. You need to upgrade <a href="https://www.mediawiki.org/wiki/Extension:Semantic_Forms">Semantic Forms</a> first.' );
+if ( isset( $wgWikimediaTravisCI ) && $wgWikimediaTravisCI == true ) {
+	if ( is_readable( __DIR__ . '/../../vendor/SemanticForms/SemanticForms.php' ) ) {
+		require_once __DIR__ . '/../../vendor/SemanticForms/SemanticForms.php';
```

Comment:
Since mediawiki core merges the packages from extensions into its own vendor folder so when travis tests the extension the vendor folder wont be created in the extension it self it will be created in mediawiki core. So for a travis specific hack lets do this. And since semanticforms is not composer ready meaning it is avalible in composer but wont be able to be loaded through composer so there wont be any psr code or anything.